gamebook/game_manager.py

```diff
@@ -0,0 +1,42 @@
+from gamebook.terminal_output import TerminalOutput
+from gamebook.terminal_input import TerminalInput
+import sys
+
+
+class GameManager(object):
+    def __init__(self, scene):
+        self.scene = scene
+        self.output_instance = TerminalOutput()
+        self.input_instance = TerminalInput()
+
+    def start(self, scene_import):
```

Comment:
Like I said in an earlier comment, change this to: `first_scene`/`initial_scene`...